src/Microsoft.ML.AutoML/Sweepers/SweeperProbabilityUtils.cs

```diff
@@ -98,13 +98,15 @@ public static float[] ParameterSetAsFloatArray(IValueGenerator[] sweepParams, Pa
                 }
                 else if (sweepParam is LongValueGenerator lvg)
                 {
+                    var longValue = GetIfIParameterValueOfT<long>(pset) ?? long.Parse(pset.ValueText);
                     // Normalizing all numeric parameters to [0,1] range.
-                    result.Add(lvg.NormalizeValue(new LongParameterValue(pset.Name, long.Parse(pset.ValueText))));
+                    result.Add(lvg.NormalizeValue(new LongParameterValue(pset.Name, longValue)));
                 }
                 else if (sweepParam is FloatValueGenerator fvg)
                 {
+                    var floatValue = GetIfIParameterValueOfT<float>(pset) ?? float.Parse(pset.ValueText);
                     // Normalizing all numeric parameters to [0,1] range.
-                    result.Add(fvg.NormalizeValue(new FloatParameterValue(pset.Name, float.Parse(pset.ValueText))));
```

Comment:
I suspect there's an upstream value being encoded with `CultureInfo.InvariantCulture`, which causes this `float.Parse(pset.ValueText)` to fail. Would parsing here with `float.Parse(pset.ValueText, CultureInfo.InvariantCulture)` suffice?

Much like a pervious PR which addressed a similar instance: https://github.com/dotnet/machinelearning/pull/4635/.

Reply:
Thanks @justinormont 
I've added parsing using `CultureInfo.InvariantCulture` in the case when parsing should happen.
Should I leave parsing in all cases or is skipping it better if we already have `IParameterValue<float>` there?